src/modules/auth/tenant/TenantDetailContainer.js

```diff
@@ -38,8 +38,6 @@ function TenantDetailContainer(props) {
 	const [page, setPage] = useState(1);
 	const [filter, setFilter] = useState("");
 	const limit = 10;
-	const [username, setUsername] = useState();
```

Comment:
@petrKavulok Ok, why were those items removed and why it is not in changelog or at least in the description of this PR then? Its very hard to distinguish from reviewers point of view, why this change has been made without any context .

Reply:
I removed it because I noticed that this functionality was already taken care of by the `<Credentials />` component. I should have made a separate PR for that. Sorry, next time I'll separate it to two PRs